MAINTAINERS.md

```diff
@@ -0,0 +1,120 @@
+# Maintainers
+
+This document serves to clarify the responsibilities of a maintainer as well as
+serve as a wiki for maintainers to link to / include resources.
+
+
+## Current Maintainers
+
+The current maintainers are:
+
+| Name  | Focus Areas |
```

Comment:
Pretty sure I already left a comment about this in the other pull, but I don't see the the point of a focus area column. The focus area is something that is entrusted to the maintainer to decide for themselves. If they can't do that, they shouldn't become a maintainer in the first place.

Also, the focus area may constantly change. For example I no longer care about functional tests as much as I did 5 years ago, because they are in a much better shape by now. Am I supposed to remove myself from that focus area now? Also, achow recently merged a p2p change in commit 22d96d76ab02fc73e7fe0d810bacee4c982df085, so is someone supposed to edit this table before or after the fact? Finally, if all rows mention "general" I wonder what the point is, since the conveyed information is minimal if not none.

Reply:
I responded on this topic in the other PR ICYMI: https://github.com/bitcoin/bitcoin/pull/25560#discussion_r945171237

Not every maintainer has General tag here. For example, @hebasto and @achow101 do not. It's up to them if they want to update it to list General or not.

If you don't care about the test system as much these days, and don't want to review the work there, it is no issue to update your focus areas. Doing so may also even help highlight if that area is under-maintained, and could maybe use a new maintainer.

W.r.t. @achow101  merging a P2P change, as noted in the document, these are not enforced rules, just documentation. If @achow101 plans to regularly work on P2P, I would think he should update his focus area, if it is one-off or infrequent maybe not. 

Perhaps it would be nice to pair with this table a list of all "focus tags" / systems, regardless of if there is a maintainer currently focusing on it?

Reply:
The details here will be effectively useless the more broad (Assume my focus area is tests, but someone creates a test for macOS, which I may not be able to review or test) they are and the less often they are updated. I can promise you that this list won't be updated, just like no one updated `REVIEWERS.md`.

Reply:
Have to agree. I'd be surprised if it even gets promptly updated when maintainers join/depart.

Reply:
Is there a lighter touch way that can be done?